Intersect.Client/Core/Main.cs

```diff
@@ -226,7 +226,7 @@ private static void ProcessGame()
                                 var map = MapInstance.Get(Globals.MapGrid[x, y]);
                                 if (map != null)
```

Comment:
And now that I think about it this could have been written as `canShowWorld &= (map?.IsLoaded ?? true)` (or maybe `&&=`? I don't remember the C# operator rules for booleans).